### ISSUE TITLE
S_REPLY_GUILD_LIST.1 not working

### ISSUE THREAD

**Issue by MuwuM** [meishuu/tera-data#241]
When I try to listen to S_REPLY_GUILD_LIST I get the following error (every time I get the response):

This was recorded today on EU Server with protocol version: 321154 

```
[00:31:27 GMT+0200] ERROR tera-proxy-game: [dispatch] handle: failed to parse S_REPLY_GUILD_LIST<1>
hook: (anonymous) in
data: e6063c6306001400010000005f100000276300001400b40136004e007800fd0600000901000001000000050000000100000041000000490020004e00200053002000410020004e002000450000006700750069006c0064006c006f0067006f005f00320038005f0031003700380039005f003200380000002600710075006f0074003b004d0061006b006500200049004e00530041004e004500200067007200650061007400200061006700610069006e0021002600710075006f0074003b000d0068007400740070003a002f002f0064006900730063006f00720064002e006d0065002f0074006500720061005f0069006e00730061006e0065000d004f006600660065006e0020006600fc0072003a0020000d002b005300740061006d006d006700720075007000700065006e003a00200028006100620020003300200053007000690065006c00650072006e0029000d002b00450069006e007a0065006c0073007000690065006c00650072003a002000470053003a0020003400330030002000200026002000700072006f006700720065007300730069006e0074006500720065007300730069006500720074000000b4017003d601ea01160279540000e6000000020000000500000001000000410000005400770069006e007400610069006c00730000006700750069006c0064006c006f0067006f005f00320038005f00320031003600320035005f003500390000002220200020005700690072002000730069006e0064002000650069006e0020006e00650074007400650072002000480061007500660065006e00200026002000480069006c00660062006500720065006900740021002000480061007500700074007300e400630068006c006900630068002000650069006e0065002000460065006900650072006100620065006e006400670069006c006400650020003b00330020000d00222020002000470049004c004400450020004f0048004e00450020005400530020002600200044004900530043004f00520044002000210020002100200021000d0022202000200041006b007a00650070007400690065007200740020006400610073002c0020006f00640065007200200062006500770065007200620074002000650075006300680020006700610072006e00690063006800740020006500720073007400200026006c0074003b00330000007003a6049203b603e00336260000b00000000100000005000000010000004100000045006d00700069007200650020006f00660020007400680065002000530075006e0000006700750069006c0064006c006f0067006f005f00320038005f0039003700380032005f003500360000005700690072002000730069006e0064002000650069006e00650020006e0065007400740065002c002000680069006c006600730062006500720065006900740065002000470069006c00640065002e0020002000540073002000690073007400200076006f007200680061006e00640065006e002e002000570069007200200073007000690065006c0065006e0020006f0068006e0065002000530074007200650073007300200075006e0064002000500066006c00690063006800740065006e002e000000a6040a05c804e0040805732200007d000000020000000500000010000000410000004c0020004500200047002000490020004f0020004e0000006700750069006c0064006c006f0067006f005f00320038005f0038003800310039005f003600000000000a0568052c053c0566054e5d00009700000002000000050000000f00000041000000420072006900670061006400610000006700750069006c0064006c006f0067006f005f00320038005f00320033003800380036005f00340000000000680500008a05aa05d4054f6900005600000001000000050000000a00000041000000490020004e0020005600200049002000430020005400200055002000530000006700750069006c0064006c006f0067006f005f00320038005f00320036003900350039005f00330000005700690072002000730069006e0064002000650069006e0020006c0065006900630068007400200062006500730063006800650075006500720074006500720020006e00650074007400650072002000680061007500660065006e002000640065007200200061006d002000610075006600620061007500200061006c006c0065006d00200061006e00660061006e0067007300200069007300740020003a00440021002000420065006900200077006500690074006500720065006e002000460072006100670065006e00200057007300700020002d002d002600670074003b00200061006e003a0020004d0069006e006b0061002c002000410070006500630074002c0020005900e20074006f000000
error: Index out of range
    at checkOffset (buffer.js:968:11)
    at Buffer.readUInt16LE (buffer.js:1014:5)
    at Readable.uint16 (\tera-data-parser\lib\protocol\stream.js:38:29)
    at Readable.string (\tera-data-parser\lib\protocol\stream.js:78:21)
    at parseField (\tera-data-parser\lib\protocol\index.js:355:37)
    at TeraProtocol.parse (\tera-data-parser\lib\protocol\index.js:364:7)
    at parseField (\tera-data-parser\lib\protocol\index.js:325:33)
    at TeraProtocol.parse (\tera-data-parser\lib\protocol\index.js:364:7)
```
Maybe it is interesting to notice that it looks like the message is cut at the end it ends with `000000` even with different search parameters:

```
[00:35:41 GMT+0200] ERROR tera-proxy-game: [dispatch] handle: failed to parse S_REPLY_GUILD_LIST<1>
hook: (anonymous) in 
data: b0023c6306001400010000000800000027630000140072003600460070004e5d00009700000002000000050000000f00000041000000420072006900670061006400610000006700750069006c0064006c006f0067006f005f00320038005f00320033003800380036005f003400000000007200fa009400b000da008a4700001400000001000000000000000100000041000000420072006900670068007400200046006c0061006d006500730000006700750069006c0064006c006f0067006f005f00320038005f00310038003300310034005f00310000004a007500730074003400460075006e002d005400770069006e006b0073000000fa0060011c0134015e019d5200000c000000010000000000000001000000410000004200720069006700680074002000530069006400650000006700750069006c0064006c006f0067006f005f00320038005f00320031003100340039005f003100000000006001060282019a01c401d9380000140000000100000000000000010000004100000053006f0073002000420072006900670061006400650000006700750069006c0064006c006f0067006f005f00320038005f00310034003500350033005f003200000041006e007a00650069006700650020006600fc007200200045007500720065002000470069006c00640065002000730063006800720065006900620065006e00000006026c02280242026a02490a0000100000000100000000000000010000004100000057006f006c0066007300620072006900670061006400650000006700750069006c0064006c006f0067006f005f00320038005f0032003600330033005f003400000000006c0200008e02ac02ae02084900000d00000001000000000000000100000041000000500061006e0074007300750020004200720069006700610064006500000000000000
error: Index out of range
    at checkOffset (buffer.js:968:11)
    at Buffer.readUInt16LE (buffer.js:1014:5)
    at Readable.uint16 (\tera-data-parser\lib\protocol\stream.js:38:29)
    at Readable.string (\tera-data-parser\lib\protocol\stream.js:78:21)
    at parseField (\tera-data-parser\lib\protocol\index.js:355:37)
    at TeraProtocol.parse (\tera-data-parser\lib\protocol\index.js:364:7)
    at parseField (\tera-data-parser\lib\protocol\index.js:325:33)
    at TeraProtocol.parse (\tera-data-parser\lib\protocol\index.js:364:7)
```

**Comment by MuwuM** [comment]
Rechecked on NA. Same error with protocol version: 321152

[19:02:13 GMT+0200] ERROR tera-proxy-game: [dispatch] handle: failed to parse S_REPLY_GUILD_LIST<1>
hook: (anonymous) in \scripts\skill-logger
data: 1e0927e30600140001000000b1120000cb71000014001402360054008400649e0000a40000000100000005000000010000004100000053006800610064006f0077002000460061006300740069006f006e0000006700750069006c0064006c006f0067006f005f0034003000300034005f00340030003500340038005f0031003100000041002000660061006d0069006c00790020006f006600200069006e0064006900760069006400750061006c0073002000640065006400690063006100740065006400200074006f002000680065006c00700069006e00670020006f006e006500200061006e006f007400680065007200200061006e00640020007300740072006900760069006e006700200066006f0072002000670072006500610074006e00650073007300210020004c006f006f006b0069006e006700200066006f007200200061006300740069007600650020006d0065006d00620065007200730020007300650065006b0069006e00670020006100200068006f006d006500200074006f00200068006100760065002000660075006e0020007700680069006c007300740020006300720065006100740069006e006700200073006f006d0065007400680069006e00670020006200690067006700650072002e0020004900740027007300200067006f006e006e0061002000740061006b006500200061006c006c0020006f006600200075007300210000001402660336024a027c02219c00002301000001000000050000000100000041000000470072006100630065006c0065007300730000006700750069006c0064006c006f0067006f005f0034003000300034005f00330039003900360039005f00320033003800000057006500200061007200650020007300650065006b0069006e006700200070006c00610079006500720073002000770068006f0020006100720065002000610063007400690076006500200061006e006400200070007200650070006100720065006400200066006f007200200065006e006400670061006d006500200063006f006e00740065006e0074002e00200048006f00700069006e006700200074006f00200072006500630072007500690074002000660075006e00200061006e00640020006500780063006900740069006e00670020006d0065006d0062006500720073007e00200000006603280588039e03ce0354a10000c6000000010000000500000001000000410000005400680065002000590061006b0075007a00610000006700750069006c0064006c006f0067006f005f0034003000300034005f00340031003300300030005f0034003900000050006c006100790065007200730020006f006600200061006e00790020006c006500760065006c002000690073002000770065006c0063006f006d0065002c002000720075006e006e0069006e006700200065006e0064002000670061006d006500200063006f006e00740065006e00740073002000610073002000610020006700750069006c006400200061006e0064002000770069006c006c0069006e006700200074006f00200074006500610063006800200061006e0064002000720075006e00200074007200610069006e0069006e0067002000720075006e00730021002000410063007400690076006500200064006900730063006f0072006400200069006600200079006f0075002000680061007600650020006f006e006500210020004c0061006e006300650072002f0042007200610077006c0065007200730020005000720065006600650072007200650064002e0000002805c4054a0552058205b1a00000de000000010000000700000001000000410000004a006f00790000006700750069006c0064006c006f0067006f005f0034003000300034005f00340031003100330037005f003300360000004b006100770061006900690020006600750072007200690065007300200061006e006400200065006c0069006e00730020006f006e006c00790020007e000d000000c4053007e6050606340618a20000af0000000100000001000000010000004100000046006c006f006f00720020004900730020004d007900200042006500640000006700750069006c0064006c006f0067006f005f0034003000300034005f00340031003400390036005f0033000000570065002000610072006500200061002000620075006e006300680020006f0066002000530045004100200070006c00610079006500720073002000770068006f00200066006f006300750073006500730020006f006e002000500056004500200061006e0064002000700072006f00760069006400650073002000670065006100720069006e0067002f0074007200610069006e0069006e0067002000720075006e007300200066006f007200200065006e0064002000670061006d0065002000640075006e00670065006f006e0073000d004a006f0069006e00200075007300200074006f0064006100790020005e002e005e002f002100000030070000520760078e0703a50000c30000000100000005000000010000004100000046006500720061006c00730000006700750069006c0064006c006f0067006f005f0034003000300034005f00340032003200340033005f003700000046006500720061006c0073002000690073002000610020006e006500770020006700750069006c00640020006c006f006f006b0069006e006700200066006f00720020006e0065007700200061006e00640020007600650072006500740061006e0020006d0065006d00620065007200730020006f006600200061006c006c0020006c006500760065006c0073002e00200057006500270072006500200061002000680065006c007000660075006c002e00200066007200690065006e0064006c00790020006700750069006c00640020006c006f006f006b0069006e006700200066006f00720020006e0065007700200066007200690065006e0064007300200061006e0064002000770065002000680061007600650020006100200064006900730063006f00720064002000730065007200760065007200200074006f006f002e0020004c006f006f006b0069006e006700200066006f0072007700610072006400200074006f00200073006500650069006e006700200079006f00750020007400680065007200650021000000
error: Index out of range
    at checkOffset (buffer.js:968:11)
    at Buffer.readUInt16LE (buffer.js:1014:5)
    at Readable.uint16 (\node_modules\tera-data-parser\lib\protocol\stream.js:38:29)
    at Readable.string (\node_modules\tera-data-parser\lib\protocol\stream.js:78:21)
    at parseField (\node_modules\tera-data-parser\lib\protocol\index.js:355:37)
    at TeraProtocol.parse (\node_modules\tera-data-parser\lib\protocol\index.js:364:7)
    at parseField (\node_modules\tera-data-parser\lib\protocol\index.js:325:33)
    at TeraProtocol.parse (\node_modules\tera-data-parser\lib\protocol\index.js:364:7)

**Comment by MuwuM** [comment]
Should be resolved by pull request [#242](https://github.com/meishuu/tera-data/pull/242)
